### PR TITLE
feat(node): support client secret

### PR DESCRIPTION
**Pull request by wangsijie**
## Summary

Add support for OIDC's `client_secret`, it is called `appSecret` here.

## Testing

Local tested.

### CODE REVIEW

File: packages/node/src/index.ts [hunk 1]
```diff
@@ -17,7 +17,22 @@ export default class LogtoClient extends BaseClient {
...
-      requester: createRequester(fetch),
+      requester: createRequester(
+        config.appSecret
+          ? async (...args: Parameters<typeof fetch>) => {
+              return fetch(args[0], {
+                ...args[1],
+                headers: {
+                  ...args[1]?.headers,
+                  authorization: `basic ${Buffer.from(
+                    `${config.appId}:${config.appSecret ?? ''}`,
```
Comment (darcyYe) [inline packages/node/src/index.ts:28]: Will config.appSecret be false as the entry condition is a truthy config.appSecret?

File: packages/node/src/index.ts [hunk 2]
```diff
@@ -17,7 +17,22 @@ export default class LogtoClient extends BaseClient {
   constructor(config: LogtoConfig, adapter: Pick<ClientAdapter, 'navigate' | 'storage'>) {
     super(config, {
       ...adapter,
-      requester: createRequester(fetch),
+      requester: createRequester(
+        config.appSecret
+          ? async (...args: Parameters<typeof fetch>) => {
+              return fetch(args[0], {
+                ...args[1],
```
Comment (demonzoo) [inline packages/node/src/index.ts:24]: What are `args[0]` and `args[1]` specifically? `url` and `configs`?
Would it be better to de-construct them first?
```ts
const [url, configs] = args;
```

File: packages/node/src/index.ts (hunk 1 above)
Comment (demonzoo) [reply packages/node/src/index.ts:28]: Would it be better to apply client secret authorization only when `appSecret` is not empty?
```
...conditional(config.appSecret && ({ authorization: `basic ${ ... }` }))
```

File: packages/node/src/index.ts (hunk 2 above)
Comment (wangsijie) [reply packages/node/src/index.ts]: updated

image

File: packages/node/src/index.ts (hunk 1 above)
Comment (wangsijie) [reply packages/node/src/index.ts:30]: in line 21:
```
config.appSecret
?
```

**Review by demonzoo — APPROVED** [review]

**Review by darcyYe — APPROVED** [review]

**Pull request merged**